YJE/Day3/두 개 뽑아서 더하기.js

```diff
@@ -0,0 +1,16 @@
+function solution(numbers) {
+    var answer = [];
+    
+    for(let i=0; i < numbers.length; i++) {
```

Comment:
```suggestion
    for(let i = 0; i < numbers.length - 1; i++) {
```
이 코드에선 문제되진 않는데 numbers 배열의 (마지막 index - 1) 까지만 반복하는게 좀 더 정확한 표현 같습니다.

Reply:
확실히 길이만큼 배열을 도는 것보다 (마지막 index - 1)로 하는게 더 효율적인 것 같네요! 코멘트 달아주셔서 감사합니다